lib/cloud_controller/permissions.rb

```diff
@@ -276,7 +267,7 @@ def readable_space_scoped_spaces_query
     end
   end
 
-  def can_read_route?(space_guid, org_guid)
+  def can_read_route?(space_guid)
```

Comment:
This method could also be changed to accept `space_id` instead.

Reply:
Agreed, though I've already made that change in another branch I'm working on at the moment, so to avoid making it in two places I'll maybe skip it here.